week2/Q2.py

```diff
@@ -0,0 +1,47 @@
+from btree import BinaryTree
+from Q1 import get_ancestors
+
+
+def find_common_ancestor(ancestors1, ancestors2):
```

Comment:
nit: This should probably be considered private. That is, it should be named `_find_common_ancestor`.